conf/routes

```diff
@@ -2,59 +2,47 @@
+GET     /v1/enterprises/$ern<\d{10}>/periods/$period<\d{4}(0[1-9]|(1[0-2]))>/legalunits/$ubrn<\d{16}>                   controllers.v1.LegalUnitController.retrieveLegalUnit(ern, period, ubrn)
+GET     /v1/enterprises/:ern/periods/:period/legalunits/:ubrn                                                           controllers.v1.LegalUnitController.badRequest(ern, period, ubrn: Option[String])
+GET     /v1/enterprises/$ern<\d{10}>/periods/$period<\d{4}(0[1-9]|(1[0-2]))>/legalunits                                 controllers.v1.LegalUnitController.retrieveAllLegalUnitsForEnterprise(ern, period)
+GET     /v1/enterprises/:ern/periods/:period/legalunits                                                                 controllers.v1.LegalUnitController.badRequest(ern, period, None: Option[String])
 
 # Reporting Units
-GET     /v1/enterprises/$ern<\d{10}>/periods/$period<\d{4}(0[1-9]|1[0-2])>/reportingunits/$rurn<\d{11}>                   controllers.v1.ReportingUnitController.retrieveReportingUnit(ern, period, rurn)
-GET     /v1/enterprises/:ern/periods/:period/reportingunits/:rurn                                                         controllers.v1.ReportingUnitController.badRequest(ern, period, rurn: Option[String])
-GET     /v1/enterprises/$ern<\d{10}>/periods/$period<\d{4}(0[1-9]|1[0-2])>/reportingunits                                 controllers.v1.ReportingUnitController.retrieveAllReportingUnitsForEnterprise(ern, period)
-GET     /v1/enterprises/:ern/periods/:period/reportingunits                                                               controllers.v1.ReportingUnitController.badRequest(ern, period, None: Option[String])
+GET     /v1/enterprises/$ern<\d{10}>/periods/$period<\d{4}(0[1-9]|1[0-2])>/reportingunits/$rurn<\d{11}>                 controllers.v1.ReportingUnitController.retrieveReportingUnit(ern, period, rurn)
+GET     /v1/enterprises/:ern/periods/:period/reportingunits/:rurn                                                       controllers.v1.ReportingUnitController.badRequest(ern, period, rurn: Option[String])
+GET     /v1/enterprises/$ern<\d{10}>/periods/$period<\d{4}(0[1-9]|1[0-2])>/reportingunits                               controllers.v1.ReportingUnitController.retrieveAllReportingUnitsForEnterprise(ern, period)
+GET     /v1/enterprises/:ern/periods/:period/reportingunits                                                             controllers.v1.ReportingUnitController.badRequest(ern, period, None: Option[String])
 
 # Other Routes
-
 GET     /                                   controllers.HomeController.status
 
 # Route for deployed version information (in JSON).
-
 GET     /version                            controllers.VersionController.version
 
 # Build dates
-
 # GET   /latest                             controllers.v1.LastUpdateController.latestListings
```

Comment:
Shall we remove /latest and the associated LastUpdateController rather than leave it commented out?

Reply:
Good spot.  Have pushed update which removes LastUpdateController.